app/screens/home/DetailsScreen.tsx

```diff
@@ -423,62 +452,112 @@ console.log(videoIsFocused,"..videoIsFocused");
+                    // source={{html: bodydata}} {...htmlProps}
+                    baseFontStyle={styles.htmlTitleCode}
+                    tagsStyles={{
+                      img: { maxWidth: Dimensions.get('window').width },
+                      p: { marginBottom: 15, marginTop: 0, textAlign: 'left' },
+                      h1: { marginBottom: 0, marginTop: 10, textAlign: 'left' },
+                      h2: { marginBottom: 15, marginTop: 0, textAlign: 'left' },
+                      h3: { marginBottom: 15, marginTop: 0, textAlign: 'left' },
+                      h4: { marginBottom: 15, marginTop: 0, textAlign: 'left' },
+                      h5: { marginBottom: 15, marginTop: 0, textAlign: 'left' },
+                      h6: { marginBottom: 15, marginTop: 0, textAlign: 'left' },
+                      span: { marginBottom: 15, marginTop: 0, textAlign: 'left' },
+                      li: { textAlign: 'left' },
+                      br: { height: 0 },
+                      iframe: { maxWidth: '100%', height: 200 }
+                    }}
+                    WebView={WebView}
+                    ignoredTags={IGNORED_TAGS}
+                  />
+                </View>
+                : null
+              }
+
+              {detailDataToUse && detailDataToUse.summary ?
+                <View style={styles.marginBottom10}>
+                  <HTML
+                    source={{ html: queryText != undefined ? queryText.length != 0 ? addSpaceToHtml(highlightWord(detailDataToUse?.summary, queryText)) : addSpaceToHtml(detailDataToUse?.summary) : addSpaceToHtml(detailDataToUse?.summary) }} key={detailDataToUse.id}
+                    // source={{html: bodydata}} {...htmlProps}
+                    baseFontStyle={styles.htmlSummaryCode}
+                    tagsStyles={{
+                      img: { maxWidth: Dimensions.get('window').width },
+                      p: { marginBottom: 15, marginTop: 0, textAlign: 'left' },
+                      h1: { marginBottom: 0, marginTop: 10, textAlign: 'left' },
+                      h2: { marginBottom: 15, marginTop: 0, textAlign: 'left' },
+                      h3: { marginBottom: 15, marginTop: 0, textAlign: 'left' },
+                      h4: { marginBottom: 15, marginTop: 0, textAlign: 'left' },
+                      h5: { marginBottom: 15, marginTop: 0, textAlign: 'left' },
+                      h6: { marginBottom: 15, marginTop: 0, textAlign: 'left' },
+                      span: { marginBottom: 15, marginTop: 0, textAlign: 'left' },
+                      li: { textAlign: 'left' },
+                      br: { height: 0 },
+                      iframe: { maxWidth: '100%', height: 200 }
+                    }}
+                    WebView={WebView}
+                    ignoredTags={IGNORED_TAGS}
+                  />
+                </View>
+                : null
+              }
+
+              {detailDataToUse && detailDataToUse.body ?
```

Comment:
I would refactor these title, summary and body if possible not to repeat similar code.